README.md

```diff
@@ -14,60 +21,54 @@ A Drupal Commerce demo store connected with a (testnet) BTCPay Server where you
 * Drupal Commerce 2.x installed ([installation guide](https://docs.drupalcommerce.org/commerce2/developer-guide/install-update/installation))  
 * Drupal: [configured and writable private file system](https://www.drupal.org/docs/8/core/modules/file/overview#content-accessing-private-files)
 
-## Installation and configuration
+## Installation and configuration Guide for BTCPay Server-Drupal Commerce Integration
```

Comment:
Installation and configuration guide for the BTCPay Server - Drupal Commerce integration